### PR TITLE
initial commit of blacklist

### PR DESCRIPTION
**Pull request by rohitaik**
create CRUD operation and write a method to save the object from byte array

### CODE REVIEW

File: core/src/main/java/in/aiknow/invio/services/NGWordsServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import java.util.List;
+import javax.inject.Inject;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONObject;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class NGWordsServiceImpl implements INGWordsService{
```
Comment (sandeepmkd) [inline core/src/main/java/in/aiknow/invio/services/NGWordsServiceImpl.java:30]: Check if the google style sheet is imported, formatting looks off

File: core/src/main/java/in/aiknow/invio/entities/BlackList.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Table(name = "black_list")
+@AllArgsConstructor
+@NoArgsConstructor
+public class BlackList extends BaseObject {
+
+  @Column(name = "first_name", columnDefinition = "text")
```
[the remark this answers is not in the document]
Comment (rohitaik) [reply core/src/main/java/in/aiknow/invio/entities/BlackList.java:19]: added.

File: core/src/main/java/in/aiknow/invio/entities/NGWords.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+@Data
+@Entity
+@Table(name = "ng_words")
+@AllArgsConstructor
+@NoArgsConstructor
+public class NGWords extends BaseObject {
+
+  @Column(name = "word", columnDefinition = "text")
+  private String word;
```
[the remark this answers is not in the document]
Comment (rohitaik) [reply core/src/main/java/in/aiknow/invio/entities/NGWords.java:20]: added column.

File: core/src/main/java/in/aiknow/invio/services/BlackListServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  @Override
+  public Page<BlackList> getAllBlackList(SearchQuery searchQuery) {
+    log.info("Get All BlackList request received");
+    Page<BlackList> results = blackListRepository
+        .findAll(SearchSpecification.build(searchQuery), searchQuery.getPageRequest());
+    return results;
+  }
+
+  @Override
+  public void AddAllBlackList(byte[] bytesArray) throws Exception {
```
[the remark this answers is not in the document]
Comment (rohitaik) [reply core/src/main/java/in/aiknow/invio/services/BlackListServiceImpl.java]: done

File: core/src/main/java/in/aiknow/invio/services/NGWordsServiceImpl.java (hunk 1 above)
Comment (rohitaik) [reply core/src/main/java/in/aiknow/invio/services/NGWordsServiceImpl.java]: formatted.

**Pull request merged**